_rules/auto-update-text-efbfc7.md

```diff
@@ -36,7 +36,7 @@ This rule applies to any [HTML element][] that has a [visible][] [text node][] a
 
 ## Expectation
 
-For each test target there is at least one set of [instruments][instrument], where each [instrument][] is in the same [web page][] as the test target or can be found in a [clearly labeled location][] from that [web page][], to achieve at least one of the following objectives:
+For each test target, the text change is a [non essential text change][] and there is at least one set of [instruments][instrument], where each [instrument][] is in the same [web page][] as the test target or can be found in a [clearly labeled location][] from that [web page][], to achieve at least one of the following objectives:
```

Comment:
My goal was actually to exclude essential text changes from the expectation, but since we can't use the definition inside of the applicability I don't really have a way to do that. I was really trying to avoid the `not a non-essential text change` language since the double negative is somewhat confusing, but I'm not sure we can avoid it.